org.eclipse.january/src/org/eclipse/january/dataset/DatasetUtils.java

```diff
@@ -32,7 +32,7 @@
 	/**
 	 * Setup the logging facilities
 	 */
-	transient protected static final Logger utilsLogger = LoggerFactory.getLogger(DatasetUtils.class);
+	transient private static final Logger utilsLogger = LoggerFactory.getLogger(DatasetUtils.class);
```

Comment:
Not sure why this is transient but I guess is unrelated.

Reply:
You can remove the `transient` qualifier